README.md

```diff
@@ -98,40 +115,38 @@ const signalFxLambda = require('signalfx-lambda');
 
 exports.handler = signalFxLambda.wrapper((event, context, callback) => {
   ...
-  signalFxLambda.helper.sendGauge('gauge.name', value);
-  callback(null, 'Done');
 });
 ```
 
-Using async/await
+2. Use async/await:
+
 ```
 'use strict';
 
 const signalFxLambda = require('signalfx-lambda');
 
 exports.handler = signalFxLambda.asyncWrapper(async (event, context) => {
   ...
-  signalFxLambda.helper.sendGauge('gauge.name', value);
 });
 ```
 
-### Deployment
+## Step 5: Deploy the function 
```

Comment:
Steps 5 - 8 are for developers who wish to test locally or from AWS (however we do not expect customers to do so).They may, if they wish, but those steps are not required for proper installation/configuration of the wrapper.

Reply:
Adding "Optional" to the beginning of the step.